Bottom: 0cb350b32de54ca673dd3831cf721dc6230c504a
Top:    e1869f35665bbd7f5551e3e7bf53cdf7437d9671
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 20:50:24 +0100

hid/gtk: Remove usage of deprecated gtk_range_set_update_policy()

We only set this to the default value anyway, and the API has been
removed with no replacements.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index cf05e4b..73bada5 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1280,8 +1280,6 @@ ghid_build_pcb_top_window (void)
   ghidgui->v_range =
     gtk_vscrollbar_new (GTK_ADJUSTMENT (ghidgui->v_adjustment));
 
-  gtk_range_set_update_policy (GTK_RANGE (ghidgui->v_range),
-			       GTK_UPDATE_CONTINUOUS);
   gtk_box_pack_start (GTK_BOX (hbox), ghidgui->v_range, FALSE, FALSE, 0);
 
   g_signal_connect (G_OBJECT (ghidgui->v_adjustment), "value_changed",
@@ -1291,8 +1289,6 @@ ghid_build_pcb_top_window (void)
 					      10.0, 10.0, 10.0);
   ghidgui->h_range =
     gtk_hscrollbar_new (GTK_ADJUSTMENT (ghidgui->h_adjustment));
-  gtk_range_set_update_policy (GTK_RANGE (ghidgui->h_range),
-			       GTK_UPDATE_CONTINUOUS);
   gtk_box_pack_start (GTK_BOX (ghidgui->vbox_middle),
                       ghidgui->h_range, FALSE, FALSE, 0);
 
diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 3be7931..7136fbb 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -479,7 +479,6 @@ ghid_range_control (GtkWidget * box, GtkWidget ** scale_res,
   gtk_scale_set_value_pos (GTK_SCALE (scale), pos);
   gtk_scale_set_draw_value (GTK_SCALE (scale), set_draw_value);
   gtk_scale_set_digits (GTK_SCALE (scale), digits);
-  gtk_range_set_update_policy (GTK_RANGE (scale), GTK_UPDATE_CONTINUOUS);
 
   /* Increments don't make sense, use -1,1 because that does closest to
      |  what I want: scroll down decrements slider value.
